loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowResultDeserializer.java

```diff
@@ -49,8 +51,17 @@
                     break;
 
                 case "synchronousComponentNum":
+                    if (cVersion.compareTo(ModuleDescriptor.Version.parse("1.2")) < 0) {
```

Comment:
You can use `JsonUtil::assertGreaterThanReferenceVersion` instead

Reply:
I just added mising assertGreaterOrEqualThanReferenceVersion and assertLessThanReferenceVersion functions.